requirements-freqai-rl.txt

```diff
@@ -3,8 +3,10 @@
 
 # Required for freqai-rl
 torch==1.13.1; python_version < '3.11'
-stable-baselines3==1.7.0; python_version < '3.11'
-sb3-contrib==1.7.0; python_version < '3.11'
+#until these branches will be released we can use this
+git+https://github.com/Farama-Foundation/Gymnasium@main
+git+https://github.com/DLR-RM/stable-baselines3@feat/gymnasium-support
+git+https://github.com/Stable-Baselines-Team/stable-baselines3-contrib@feat/gymnasium-support
```

Comment:
I've marked that PR as "WIP" for now. It'll not prevent people from using it / testing this - but we'll not be adding dependencies which MUST be installed directly from github to actually work.

Once the versions are merged (and released) - please update this PR to the pinned pypi versions that include this support.

Reply:
Of course, I thought the same!

Reply:
This should actually now be available through `sb3_contrib>=2.0.0a1` - according to https://github.com/DLR-RM/stable-baselines3/issues/871.

in my understanding, this would mean we can remove all of the above git installs ... 

(Updating this is now becomming crucial, as the lastest pip version refuses to install gym==0.21 - causing all CI to fail.

Reply:
I will test thoose and let you know the results and modify this PR!

Reply:
Updated the questionable lines, tested, working perfectly with the changes of this PR!